src/applications/facility-locator/components/FacilityTypeDescription.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { facilityTypes } from '../config';
+
+class FacilityTypeDescription extends Component {
+  renderFacilityType() {
+    const { facility } = this.props;
+    const { facilityType, classification } = facility.attributes;
```

Comment:
Since you're not specifically using `facility`, you can destructure everything in one line:

```
const { facilityType, classification } = this.props.facility.attributes;
```